inc/saihostif.h

```diff
@@ -413,6 +413,18 @@ typedef enum _sai_hostif_trap_type_t
      */
     SAI_HOSTIF_TRAP_TYPE_P4RT = 0x0000400b,
 
+    /**
+     * @brief NTPCLIENT traffic (UDP/TCP src port == 123)
+     * to local router IP address (default packet action is drop)
+     */
+    SAI_HOSTIF_TRAP_TYPE_NTPCLIENT = 0x0000400c,
```

Comment:
Do we need both TCP and UDP port 123 or just UDP 123 for NTP?

Reply:
IANA lists port 123 registered for NTP over both TCP and UDP. 
https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml?search=ntp